packages/amplify-cli/tests/amplify-helpers/headless-init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Which fields out of these are optional and which are not? How do we check/validate that in our code?